### PR TITLE
Update model-notifications

### PR DESCRIPTION
**Pull request by joellabes**
## What are you changing in this pull request and why?
- The main thing I wanted to do was clarify that you can't define a new group in `dbt_project.yml`. 
- I also removed the Slack handle since we don't currently support Slack notifications and I thought that might be confusing (I also don't know whether we'll capture channel name or channel ID when we do support Slack, so didn't want people to pre-populate the wrong information). 
- After that, I also provided sample code for assigning groups to models in dbt_project.yml

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-joellabes-patch-4-dbt-labs.vercel.app/docs/deploy/model-notifications

### CODE REVIEW

File: website/docs/docs/deploy/model-notifications.md [hunk 1]
```diff
@@ -33,7 +33,9 @@ Create configuration YAML files in your project for dbt to send notifications ab
 
 ## Configure groups
 
-Add your group configuration in either the `dbt_project.yml` or `groups.yml` file. For example: 
+Define your groups in any .yml file in your [models directory](/reference/project-configs/model-paths). For example:
```
Comment (joellabes) [inline website/docs/docs/deploy/model-notifications.md:36]: This is wrong and is the main thing I wanted to do in the PR

File: website/docs/docs/deploy/model-notifications.md [hunk 2]
```diff
@@ -42,22 +44,26 @@ groups:
...
-      slack: finance-data
+      favorite_food: donuts
 
   - name: marketing
     description: "Models related to the marketing department"
     owner:
       name: "Marketing Team"
       email: marketing@dbtlabs.com
-      slack: marketing-data
+      favorite_food: jaffles
```
Comment (joellabes) [inline website/docs/docs/deploy/model-notifications.md:57]: This is for clarity and is optional, but reduces the likelihood people misbelieve that Slack notifs are supported

File: website/docs/docs/deploy/model-notifications.md [hunk 3]
```diff
@@ -42,22 +44,26 @@ groups:
...
 
-## Set up models
+</File>
+
+## Attach groups to models
+
+Attach groups to models as you would any other config, in either the `dbt_project.yml` or `<whatever>.yml` files. For example: 
 
-Set up your model configuration in either the `dbt_project.yml` or `groups.yml` file; doing this automatically sets up notifications for tests, too. For example: 
+<File name='models/marts.yml'>
```
Comment (joellabes) [inline website/docs/docs/deploy/model-notifications.md]: The rest of this is purely additive and you can rework as you see fit

File: website/docs/docs/deploy/model-notifications.md [hunk 4]
```diff
@@ -74,6 +80,32 @@ models:
...
+    marts:
+      sales:
+        +group: finance
+      campaigns:
+        +group: marketing
+    
+```
+
+</File>
+Attaching a group to a model also encompasses its tests, so you will also receive notifications for a model's test failures.
```
Comment (joellabes) [inline website/docs/docs/deploy/model-notifications.md]: As is this

File: website/docs/docs/deploy/model-notifications.md (hunk 2 above)
Comment (mirnawong1) [reply website/docs/docs/deploy/model-notifications.md:57]: good callout! i assumed slack was supported when reading

File: website/docs/docs/deploy/model-notifications.md (hunk 3 above)
Comment (mirnawong1) [reply website/docs/docs/deploy/model-notifications.md]: thanks for adding the files tag! 🙏

File: website/docs/docs/deploy/model-notifications.md [hunk 5]
```diff
@@ -42,22 +44,26 @@ groups:
...
-      slack: marketing-data
+      favorite_food: jaffles
 ```
 
-## Set up models
+</File>
+
+## Attach groups to models
+
+Attach groups to models as you would any other config, in either the `dbt_project.yml` or `<whatever>.yml` files. For example:
```
Comment (mirnawong1) [inline website/docs/docs/deploy/model-notifications.md]:
```suggestion
Attach groups to models as you would any other config, in either the `dbt_project.yml` or `whatever.yml` files. For example: 
```

File: website/docs/docs/deploy/model-notifications.md [hunk 6]
```diff
@@ -74,6 +80,32 @@ models:
       group: marketing
 
 ```
+</File>
+
+By assigning groups in the `dbt_project.yml` file, you can capture all models in a subdirectory at once. In the below example, all notifications related to staging models will go to the data engineering group, whereas models in `marts/sales` will be routed to the finance team.
```
Comment (mirnawong1) [inline website/docs/docs/deploy/model-notifications.md]:
```suggestion
By assigning groups in the `dbt_project.yml` file, you can capture all models in a subdirectory at once. 

In this example, model notifications related to staging models go to the data engineering group, `marts/sales` models to the finance team, and `marts/campaigns` models to the marketing team.
```

**Review by mirnawong1 — APPROVED** [review]
looks great to me and thanks for adjusting this @joellabes , made some small tweaks for clarity and merging for you!

**Pull request merged**